travis-ci/push-csharp-sdk.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+DIR=$(dirname "$0")
+DATE=`date +%Y-%m-%d:%H:%M:%S`
+
+# only push to sdk repo when it's not from pull request
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ];
+then
+    openssl aes-256-cbc -K $encrypted_b0a304ce21a6_key -iv $encrypted_b0a304ce21a6_iv -in $DIR/csharp-repo.enc -out $DIR/csharp-repo.pem -d
+    eval "$(ssh-agent -s)" #start the ssh agent
+    chmod 600 $DIR/csharp-repo.pem # this key should have push access
+    ssh-add $DIR/csharp-repo.pem
+
+    cd $DIR/../swagger-out/csharp
+    git init
```

Comment:
Would it be better to clone first instead of initializing? That way we could always have a comparison view of the travis branch compared to master here: https://github.com/square/connect-csharp-sdk/compare/travis I believe.